### ISSUE TITLE
each variable should have comment

### ISSUE THREAD

**Issue by inotetw** [realm/SwiftLint#5496]
## New rule request

Because most coders are not proficient in English, each variable should have a comment for better maintenance.

### Bad
```
var nextReloadPath: IndexPath
```

### Good
```
/// next IndexPath for reload Table
var nextReloadPath: IndexPath
```

**Comment by SimplyDanny** [comment]
Which kind of variables are you referring to? There is the `missing_docs` already which checks that all properties with a certain ACL are documented.

**Comment by inotetw** [comment]
For example, resendEmail must have documentation such as 'Resend email to user,' but missing_docs is not working for this declaration.

class SendMailOrOtp {

    struct Output {
          let resendEmail: AnyPublisher<Void, Never>
      }

}

**Comment by inotetw** [comment]
> Which kind of variables are you referring to? There is the `missing_docs` already which checks that all properties with a certain ACL are documented.

after add warning: [open, public, internal, private], my problem is solved, thanks for your help ^^

**Comment by inotetw** [comment]
I have another case where the variable 'inputData' does not trigger a warning that requires me to write a comment for the variable.

final class ViewModel: BaseViewModel {

    struct InputData {
         var  id: String?
    }

    struct Output {
        let inputData: AnyPublisher<InputData, Never>
    }
}
